exploring.rst

```diff
@@ -63,7 +63,6 @@ building your understanding of both the 2.x and 3.x versions of CPython:
     "`A guide from parser to objects, observed using GDB`_", "Code walk from Parser, AST, Sym Table and Objects", Louie Lu, 3.7.a0
     "`Green Tree Snakes`_", "The missing Python AST docs", Thomas Kluyver, 3.6
     "`Yet another guided tour of CPython`_", "A guide for how CPython REPL works", Guido van Rossum, 3.5
-    "`Python Asynchronous I/O Walkthrough`_", "How CPython async I/O, generator and coroutine works", Philip Guo, 3.5
```

Comment:
I've removed the link to the article because:

1. It returns a 404.
2. The author [mentioned](https://pg.ucsd.edu/index.html#faq) about not linking to his missing articles.